src/datapilot/utils/utils.py

```diff
@@ -40,3 +42,42 @@ def get_dir_path(path: str) -> str:
     :return:
     """
     return Path(path).parent
+
+
+# Will need to change this
+base_url = "http://localhost:5001"
```

Comment:
Needs to be injectible from the cli / environment. Let's keep app.myaltimate as default and we should inject this from somewhere (Optional cli arg or environment variable)